lms/templates/shoppingcart/receipt.html

```diff
@@ -112,7 +112,7 @@ <h3 class="text-center">${_("Please send each professional one of these unique r
                     % if order.company_name:
                       ${order.company_name}
                     % else:
-                      N/A
+                      ${_('N/A')}
                     % endif
                   </label>
```

Comment:
````<label>``` elements are reserved for use with form inputs.  They cannot be used unless they are programmatically associated with form fields.  Here is a list of the only elements ```<label>``` s can be associated with:
http://www.w3.org/html/wg/drafts/html/CR/forms.html#category-label